### PR TITLE
Remove a slash in path to save the file correctly on Linux

### PR DESCRIPTION
**Pull request by Erfaniaa**
### Describe your change:

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by CaedenPH — APPROVED** [review]
Looks good - Great spot

**Review by CaedenPH — APPROVED** [review]
Looks good - Great spot

File: computer_vision/flip_augmentation.py [hunk 1]
```diff
@@ -32,13 +32,13 @@ def main() -> None:
         letter_code = random_chars(32)
         file_name = paths[index].split(os.sep)[-1].rsplit(".", 1)[0]
         file_root = f"{OUTPUT_DIR}/{file_name}_FLIP_{letter_code}"
```
Comment (itsamirhn) [inline computer_vision/flip_augmentation.py:34]:
```suggestion
        file_root = os.path.join(OUTPUT_DIR, f"{file_name}_FLIP_{letter_code}")
```

Comment (cclauss) [reply computer_vision/flip_augmentation.py:34]: Can we use `pathlib` instead?

Comment (tianyizheng02) [reply computer_vision/flip_augmentation.py:34]:
```suggestion
        file_root = Path(OUTPUT_DIR, f"{file_name}_FLIP_{letter_code}")
```

**Review by tianyizheng02 — CHANGES_REQUESTED** [review]

**Review by cclauss — APPROVED** [review]

**Pull request merged**